mindsdb/utilities/cache.py

```diff
@@ -1,13 +1,17 @@
-import shelve
 import os
+import shelve
+import json
 
+import walrus
 from mindsdb.utilities.config import Config
 
+CONFIG = Config()
+
 
-class Cache:
+class LocalCache:
```

Comment:
There should be a common interface between this and `RedisCache`

Reply:
Done